enas/cifar10/micro_child.py

```diff
@@ -245,12 +245,13 @@ def _model(self, images, is_training, reuse=False):
 
     with tf.variable_scope(self.name, reuse=reuse):
       # the first two inputs
+      inp_c = self._get_C(images)
       with tf.variable_scope("stem_conv"):
-        w = create_weight("w", [3, 3, 3, self.out_filters * 3])
+        w = create_weight("w", [inp_c, inp_c, inp_c, self.out_filters * 3])
```

Comment:
inp_c might benefit from some other variable name, I have no idea what it means. input channels? maybe just call it `input_channels`?